### DISCUSSION TITLE
Fine Tuning Whisper CLI

### DISCUSSION THREAD

**Discussion by Zigunov** [openai/whisper#2339] [General]
I've unfortunately just wasted a day searching for fine-tuning whisper options that are compatible with the CLI version of whisper as I'm looking to implement my speech style and also retain word-level timestamps, but I haven't found a way to do both, only one or the other, unfortunately. 

If anyone can help me out with this I would appreciate it a lot. I'm looking to either train a new model for CLI or train an existing model that I have locally (small, preferably), but I have no idea where to look.

**Comment by itaipee** [comment]
can you detail what exactly is the issue: 
Is the fine-tuning itself not working ?   
Or once you did the FT , did you get model that can be used in python command , but not in command line ? 

for the first option , i suggest follow up this basic guide  https://huggingface.co/blog/fine-tune-whisper 

for the 2nd problem , use  ct2-transformers-converter    
  ct2-transformers-converter --model $ft_model --output_dir ${model}_converted --copy_files preprocessor_config.json
